Changelog.md

```diff
@@ -25,8 +25,40 @@ Language Features:
 Compiler Features:
  * Type Checker: Show named argument in case of error.
 
+Breaking changes already present in earlier releases via the "experimental 0.5.0" mode, which are not optional anymore (sorted by importance):
```

Comment:
Why not place this after the "breaking changes" block and have it sorted alphabetically (as the rest)?